dexios/src/subcommands/decrypt.rs

```diff
@@ -206,10 +206,8 @@ pub fn stream_mode(input: &str, output: &str, params: &CryptoParams) -> Result<(
             })?;
 
             let mut master_key = [0u8; 32];
-
-            for (i, byte) in master_key_decrypted.iter().enumerate() {
-                master_key[i] = *byte;
-            }
+            let len = 32.min(master_key_decrypted.len());
```

Comment:
I'm not too sure what this line does, please could you explain it?

Why would 32 not work on it's own? Thank you!

Reply:
I'm reinsuring) a panic in the CLI would not be particularly pleasant...
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=c101f4fd89ef4e5a25618a7038dc4244

Reply:
Makes sense! Thank you :)